docs/src/pages/reference/useMutation.md

```diff
@@ -39,6 +40,9 @@ mutate(variables, {
   - **Required**
   - A function that performs an asynchronous task and returns a promise.
   - `variables` is an object that `mutate` will pass to your `mutationFn`
+- `cacheTime: number | Infinity`
```

Comment:
Should we mention a default here is Infinity?

Reply:
The default is 5 minutes, like it is for queries.

Reply:
oh got it, thanks!